### PR TITLE
Update cs1614.md

### PR DESCRIPTION
**Pull request by ghost**
Description update.

### CODE REVIEW

File: docs/csharp/language-reference/compiler-messages/cs1614.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ helpviewer_keywords:
 ms.assetid: 787eef18-74a6-49d9-9f34-f0881aca8dff
 ---
 # Compiler Error CS1614
-'name' is ambiguous; between 'attribute1' and 'attribute2'. use either '@attribute' or 'attributeAttribute'  
+Description: 'name' is ambiguous between 'name' and 'nameAttribute'; use either '@name' or 'nameAttribute'.
```
Comment (pkulikov) [inline docs/csharp/language-reference/compiler-messages/cs1614.md]: @johnnyagerard please remove "Description: " at the start of the paragraph as it's not the part of the error message. Typically, the first paragraph of a compiler error article is the citation of the corresponding error message.

Comment (johnnyagerard) [reply docs/csharp/language-reference/compiler-messages/cs1614.md]: @pkulikov Alright.

**Comment by BillWagner** [comment]
@johnnyagerard I'm going to close this without accepting it.
The original text matches the compiler output. That output should be in sync with the compiler.

**Comment by johnnyagerard** [comment]
@BillWagner Can you please review this PR again?
There are errors in the description.
Original text:
>'name' is ambiguous; between 'attribute1' and 'attribute2'. use either '@attribute' or 'attributeAttribute'

Actual compiler error (from the example):
>'MySpecial' is ambiguous between 'MySpecial' and 'MySpecialAttribute'; use either '@MySpecial' or 'MySpecialAttribute'

We should at least replace the original text with the following:
>'name' is ambiguous between 'attribute1' and 'attribute2'; use either '@attribute' or 'attributeAttribute'

**Review by BillWagner — COMMENTED** [review]
This is nicely clarified @johnnyagerard  The changes match a real example 

I'll :shipit: now.  Thanks for helping to work through everything.

**Review by BillWagner — APPROVED** [review]

**Pull request merged**